content/documentation/getting-started/index.adoc

```diff
@@ -224,6 +225,50 @@ To remove the 'kiali' role from the user 'developer' in the 'myproject' namespac
 oc adm policy remove-role-from-user kiali developer -n myproject
 ----
 
+=== Namespace Management
+
+==== Accessible Namespaces
+
+The Kiali custom resource (CR) tells the Kiali Operator which namespaces are accessible to Kiali. It is specified in the CR via the `accessible_namespaces` setting.
+
+The specified namespaces are those that have service mesh components that are to be observed by Kiali.  Additionally, the namespace to which Kiali is installed must be accessible (typically the same namespace as Istio).  Each list entry can be a regex matched against all namespaces the operator can see.  If not set the default makes all namespaces accessible except for some internal namespaces that should typically be ignored.
+
+As an example, if Kiali is to be installed in the istio-system namespace, and is expected to monitor all namespaces prefixed with `mycorp_` the setting would be:
+
+[source,yaml]
+----
+deployment:
+  accessible_namespaces:
+  - istio-system
+  - mycorp_.*
+----
+
+icon:lightbulb[size=1x]{nbsp} If `accessible_namespaces` has an entry with the special value of "**" (two asterisks), it denotes that Kiali be given access to all namespaces via a single cluster role (if using this special value of "**", you are required to have already granted the operator permissions to create cluster roles and cluster role bindings). It is not recommended for production but the following command will create both the Kiali operator and the Kiali CR, configured for full cluster access to current and new namespaces:
+[source,bash]
+----
+bash <(curl -L https://git.io/getLatestKialiOperator) -an '**'
+----
+
+
+==== Excluded Namespaces
+
+The Kiali custom resource (CR) tells the Kiali Operator which accessible namespaces should be excluded from the list of namespaces provided by the API and UI. This can be useful if wildcards are used when specifying link:#_accessible_namespaces[Accessible Namespaces].  This setting has no effect on namespace accessibility.  It is is only a filter, not security-related.
```

Comment:
```suggestion
The Kiali custom resource (CR) tells the Kiali Operator which accessible namespaces should be excluded from the list of namespaces provided by the API and UI. This can be useful if wildcards are used when specifying link:#_accessible_namespaces[Accessible Namespaces].  This setting has no effect on namespace accessibility.  It is only a filter, not security-related.
```